app/src/main/java/io/rapidpro/androidchannel/RapidPro.java

```diff
@@ -76,6 +88,8 @@
     public static long MESSAGE_THROTTLE_WINDOW = 1000 * 60 * (MESSAGE_THROTTLE_MINUTES + 2);
     public static final long MESSAGE_RATE_LIMITER = 1000;
 
+    public static final String MESSAGE_PACK_VERSION_SUPPORTED = "1.1";
```

Comment:
Since every time the message packs version change the app will have to change too, I hard coded the packs version so we pass that to the server to get that proper packs version